cirq-google/cirq_google/engine/engine_program.py

```diff
@@ -615,18 +442,6 @@ def _deserialize_program(code: any_pb2.Any, program_num: Optional[int] = None) -
         raise ValueError('deserializing a v1 Program is not supported')
     elif code_type == 'cirq.google.api.v2.Program' or code_type == 'cirq.api.google.v2.Program':
         program = v2.program_pb2.Program.FromString(code.value)
-    elif code_type == 'cirq.google.api.v2.BatchProgram':
-        if program_num is None:
```

Comment:
Can we also remove the `program_num` argument from this function now that it's no longer used?